Modules/dnnGlimpse/dnnGlimpse.dnn

```diff
@@ -139,7 +144,7 @@
                       </runtimePolicies>
                     </glimpse>
                   </node>
-                  <node path="/configuration/glimpse" action="update" targetpath="/configuration/glimpse/tabs" collision="ignore">
+                  <node path="/configuration/glimpse" action="update" targetpath="/configuration/glimpse/tabs" collision="overwrite">
```

Comment:
Why the change to `overwrite`?

Reply:
If you reinstall, it adds it a second time, crashing the site.